### PR TITLE
adds license information for DailyDialog.

### PR DESCRIPTION
**Pull request by aditya2211**

### CODE REVIEW

**Comment by julien-c** [comment]
Thanks! Can you also add it as metadata in the YAML block at the top of the file?

Should be in the form:

```
licenses:
- cc-by-sa-4.0
```

**Comment by aditya2211** [comment]
seems like we need to add all the other tags ? 
```       
if error_messages:
>           raise ValueError("\n".join(error_messages))
E           ValueError: The following issues have been found in the dataset cards:
E           YAML tags:
E           __init__() missing 8 required positional arguments: 'annotations_creators', 'language_creators', 'languages', 'multilinguality', 'size_categories', 'source_datasets', 'task_categories', and 'task_ids'
```

**Comment by julien-c** [comment]
I'll let @lhoestq or @yjernite chime in (and maybe complete/merge). Thanks!

File: datasets/daily_dialog/dataset_infos.json
Comment (lhoestq) [inline datasets/daily_dialog/dataset_infos.json:1]:
```suggestion
{"default": {"description": "We develop a high-quality multi-turn dialog dataset, DailyDialog, which is intriguing in several aspects. \nThe language is human-written and less noisy. The dialogues in the dataset reflect our daily communication way \nand cover various topics about our daily life. We also manually label the developed dataset with communication \nintention and emotion information. Then, we evaluate existing approaches on DailyDialog dataset and hope it \nbenefit the research field of dialog systems.\n", "citation": "@InProceedings{li2017dailydialog,\n    author = {Li, Yanran and Su, Hui and Shen, Xiaoyu and Li, Wenjie and Cao, Ziqiang and Niu, Shuzi},\n    title = {DailyDialog: A Manually Labelled Multi-turn Dialogue Dataset},\n    booktitle = {Proceedings of The 8th International Joint Conference on Natural Language Processing (IJCNLP 2017)},\n    year = {2017}\n}\n", "homepage": "http://yanran.li/dailydialog", "license": "cc-by-nc-sa-4.0", "features": {"dialog": {"feature": {"dtype": "string", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "act": {"feature": {"num_classes": 5, "names": ["__dummy__", "inform", "question", "directive", "commissive"], "names_file": null, "id": null, "_type": "ClassLabel"}, "length": -1, "id": null, "_type": "Sequence"}, "emotion": {"feature": {"num_classes": 7, "names": ["no emotion", "anger", "disgust", "fear", "happiness", "sadness", "surprise"], "names_file": null, "id": null, "_type": "ClassLabel"}, "length": -1, "id": null, "_type": "Sequence"}}, "post_processed": {"features": null, "resources_checksums": {"train": {}, "test": {}, "validation": {}}}, "supervised_keys": null, "builder_name": "daily_dialog", "config_name": "default", "version": {"version_str": "1.0.0", "description": null, "datasets_version_to_prepare": null, "major": 1, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 7296715, "num_examples": 11118, "dataset_name": "daily_dialog"}, "test": {"name": "test", "num_bytes": 655844, "num_examples": 1000, "dataset_name": "daily_dialog"}, "validation": {"name": "validation", "num_bytes": 673943, "num_examples": 1000, "dataset_name": "daily_dialog"}}, "download_checksums": {"http://yanran.li/files/ijcnlp_dailydialog.zip": {"num_bytes": 4475921, "checksum": "c641e88cbf21fd7c1b57289387f9107d33fe8685a2b37fe8066b82776535ea89"}}, "download_size": 4475921, "post_processing_size": 0, "dataset_size": 8626502, "size_in_bytes": 13102423}}
```
cc-by-sa-4.0 -> cc-by-nc-sa-4.0

File: datasets/daily_dialog/README.md
Comment (lhoestq) [inline datasets/daily_dialog/README.md:4]:
```suggestion
annotations_creators:
- expert-generated
language_creators:
- found
languages:
- en
licenses:
- cc-by-nc-sa-4.0
multilinguality:
- monolingual
size_categories:
- 10K<n<100K
source_datasets:
- original
task_categories:
- text-classification
task_ids:
- text-classification-other-emotion-classification
```

**Review by lhoestq — APPROVED** [review]
Thanks :) I just added the missing tags

**Comment by lhoestq** [comment]
Looks like CircleCI has an incident. Let's wait for it to be working again and make sure the CI is green

**Comment by lhoestq** [comment]
The remaining error is unrelated to this PR, merging

**Pull request merged**
